pkg/jx/cmd/create_addon_flagger.go

```diff
@@ -120,5 +126,44 @@ func (o *CreateAddonFlaggerOptions) Run() error {
 			return errors.Wrap(err, fmt.Sprintf("error enabling Istio in namespace %s", ns))
 		}
 	}
+
+	// Create the Istio gateway
+	if o.IstioGateway != "" {
+		istioClient, err := o.factory.CreateIstioClient()
```

Comment:
Please create an adapter method in CommonOptions e.g. `IstioClient`. The factory is not supposed to be accessible anymore directly from cmd package. We are in the process of refactoring the cmd, and the CommonOptions will be moved in a separate package.